core/workspace_svg.ts

```diff
@@ -2715,6 +2715,7 @@ export class WorkspaceSvg
 
   /** See IFocusableTree.lookUpFocusableNode. */
   lookUpFocusableNode(id: string): IFocusableNode | null {
+    if (!id) return null;
```

Comment:
I noticed when logging there's a bunch of times that this function is triggered with an empty string for the id, which will never match anything so there's just a bunch of wasted work getting all the blocks and comments and connections and whatnot.